generate.go

```diff
@@ -40,6 +40,10 @@ var generateFlags = []cli.Flag{
 	cli.StringSliceFlag{Name: "poststart", Usage: "path to poststart hooks"},
 	cli.StringSliceFlag{Name: "poststop", Usage: "path to poststop hooks"},
 	cli.StringFlag{Name: "root-propagation", Usage: "mount propagation for root"},
+	cli.StringFlag{Name: "version", Value: "0.2.0", Usage: "version of the specification"},
```

Comment:
We need to come up with some code to support different versions of the spec here so I would leave this out of this PR for now.